scss/message/_message.scss

```diff
@@ -145,7 +145,6 @@
         object-fit: cover;
         width: auto;
         max-width: 100%;
-        height: 400px;
```

Comment:
that was always overwritten by the max height `.module-message__img-attachment` so I removed it.